### DISCUSSION TITLE
在预览版1.12.9黑夜模式中的专栏

### DISCUSSION THREAD

**Discussion by BaiXianMoYun** [the1812/Bilibili-Evolved#1956] [综合讨论 / General]
![WF }J374R`CJ_GNPVT~QE(S](https://user-images.githubusercontent.com/75004485/122219077-a9671280-cee1-11eb-8cd0-f44229e47ebf.png)
![@97_6{%AT`PWI%@D LVQS~5](https://user-images.githubusercontent.com/75004485/122219096-ae2bc680-cee1-11eb-86f7-e550de5fae4d.png)
麻了

**Comment by BaiXianMoYun** [comment]
![WF }J374R`CJ_GNPVT~QE(S](https://user-images.githubusercontent.com/75004485/122219422-ff3bba80-cee1-11eb-9e93-41568ba87779.png)
bu
![@97_6{%AT`PWI%@D LVQS~5](https://user-images.githubusercontent.com/75004485/122219441-02cf4180-cee2-11eb-9d45-160fc16a00e3.png)
补上

**Reply by Geeyun-JY3** [reply]
这张图发重了……上面 body 里那张就上传成功了，你发送前能看到相应 Markdown 语法的包裹相应图片直链的。
而且如果上传失败你可以编辑 body 然后再上传的。

**Comment by BaiXianMoYun** [comment]
![WF }J374R`CJ_GNPVT~QE(S](https://user-images.githubusercontent.com/75004485/122219520-18dd0200-cee2-11eb-9f28-27636af6b703.png)
奇怪，这张怎么发不出来

**Reply by BaiXianMoYun** [reply]
好了

**Comment by Geeyun-JY3** [comment]
麻之前能不能先看看 issue 的置顶，不要开重复的主题：#1933。
而且这属于 bug，应该去开 issue 而不是 discussion。

再而且，最新的预览离线版已经修了：#1954
![image](https://user-images.githubusercontent.com/49396421/122344344-4628bf00-cf79-11eb-8083-23dea229ec28.png)

**Reply by BaiXianMoYun** [reply]
> 麻之前不能先看问题的置顶，不要开重复的主题：#1933。
> 而且这属于bug，应该去开问题而不是讨论。
> 
> 再并且，最新的离线版已经修改了：#1954
> ![图像](https://user-images.githubusercontent.com/49396421/122344344-4628bf00-cf79-11eb-8083-23dea229ec28.png)

因为已经有一个了，所以我才开的讨论。
而且我个人觉得这可能是我自己的问题，小小调下就好了，所以没必要再去发一个问题。
然后现在已经好了，谢谢。
